updater/scripts/repository/github-api-request-types-by-count.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#
+# Count number of API calls per route
+#
+
+echo -e "type\tcount"
+
+zcat -f /var/log/github/unicorn.log.1* |
+    grep -F 'request_category=api' |
+    grep -Fv 'remote_address=127.0.0.1' |
+    grep -Fv "controller=\"Api::Internal" |
+    grep "repo=$REPOSITORY " |
+    grep -oP 'route=\K\S+' |
+    grep -Fvx 'nil' |
+    sort |
+    uniq -ic |
+    sort -rn |
+    head -20 |
+    awk '{ printf("%s\t%s\n", $2, $1) }'
```

Comment:
That looks pretty similar to https://github.com/Autodesk/hubble/blob/master/updater/scripts/api-requests-by-user.sh ... should we add a repo option to the original script to avoid code duplication?

Reply:
Good point! There are other reports I wanted to add for repo metrics that are pretty much the same as well. I'll explore adding a repo option to the original scripts.

Reply:
@larsxschneider I added a repo option for [`git-download.sh`](https://github.com/Autodesk/hubble/pull/177/files#diff-d6a56292da0a23d8f78e573bd152f501). Let me know if you think that's a good solution and if it's ok to use for other reports/scripts as well.